### PR TITLE
feat(EST-71): redesign notification component, add content-prefix

### PR DESCRIPTION
**Pull request by Seeru-crypto**

### CODE REVIEW

File: libs/styles/src/lib/scss/components/_notification.scss [hunk 1]
```diff
@@ -65,15 +69,20 @@
...
     }
 
     #{$base}__close-button {
-      position: absolute;
-      top: 20px;
-      right: 20px;
+      display: flex;
       cursor: pointer;
+
+      &.compact {
```
Comment (certainlyakey) [inline libs/styles/src/lib/scss/components/_notification.scss:80]: No need to add a modifier class for this button specifically. You can replace it with `#{$base}--size-compact #{$base}__close-button` instead.

File: libs/ui/src/lib/notification/notification.component.html [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
   <button *ngIf="showCloseButton && size !== 'compact'"
           aria-label="Close"
           class="cvi-notification__close-button"
           (click)="close()"
           type="button">
-    <cvi-ng-icon name="close" class="cvi-notification__close-button-inner" svgClass="cvi-notification__close-button-icon" dataAttribute="close-icon" [height]="16"></cvi-ng-icon>
+    <cvi-ng-icon name="close" class="cvi-notification__close-button-inner"
+                 svgClass="cvi-notification__close-button-icon" dataAttribute="close-icon" [height]="16"></cvi-ng-icon>
+  </button>
+  <button *ngIf="showCloseButton && size === 'compact'"
```
Comment (certainlyakey) [inline libs/ui/src/lib/notification/notification.component.html:11]: No need to have another copy of the close button in the template – the different styling of the "compact" close button can be simply done through CSS ancestor selector. See the other comment.

File: libs/ui/src/lib/notification/notification.component.html [hunk 3]
```diff
@@ -19,14 +30,17 @@ <h2 class="cvi-notification__title">{{ title }}</h2>
...
+                             iconClass="cvi-notification__content-icon" dataAttribute="icon">
           <ng-container *ngTemplateOutlet="content"></ng-container>
         </cvi-ng-labeled-icon>
       </ng-container>
+
       <ng-container *ngIf="size !== 'compact' || !getIconName()">
         <ng-container *ngTemplateOutlet="content"></ng-container>
       </ng-container>
       <ng-template #content>
+        <b *ngIf=contentPrefix> {{contentPrefix}}: </b>
```
Comment (certainlyakey) [inline libs/ui/src/lib/notification/notification.component.html:43]: @Seeru-crypto Should it be a separate prop? Or maybe it makes sense to insert `title` prop value here?

**Review by certainlyakey — CHANGES_REQUESTED** [review]

File: libs/styles/src/lib/scss/components/_notification.scss (hunk 1 above)
Comment (Seeru-crypto) [reply libs/styles/src/lib/scss/components/_notification.scss:80]: So if compact and regular variants have different stylings would the css look something like this?
```css
#{$base}__close-button {
// some regular style
    }

#{$base}--size-compact #{$base}__close-button {
// some compact style
    }
```

File: libs/ui/src/lib/notification/notification.component.html (hunk 3 above)
Comment (Seeru-crypto) [reply libs/ui/src/lib/notification/notification.component.html:43]: would you elaborate a bit more?
The message prefix is a bit different from the standard title, which is used here. 
Therefore I created a separate prop for it.

File: libs/styles/src/lib/scss/components/_notification.scss (hunk 1 above)
Comment (certainlyakey) [reply libs/styles/src/lib/scss/components/_notification.scss:80]: @Seeru-crypto yes, exactly!

File: libs/ui/src/lib/notification/notification.component.html (hunk 3 above)
Comment (certainlyakey) [reply libs/ui/src/lib/notification/notification.component.html:43]: > The message prefix is a bit different from the standard title, which is used here.
If that's the case, then feel free to go with the separate prop.

**Comment by Seeru-crypto** [comment]
feature is WIP, but mostly done

**Review by certainlyakey — CHANGES_REQUESTED** [review]

**Pull request closed without merging**